Hatchling is not available on Solaris. This patch modifies
the build metadata to use setuptools instead.

Not intended for upstream.

--- dnspython-2.6.1/pyproject.toml
+++ dnspython-2.6.1/pyproject.toml
@@ -1,12 +1,12 @@
 [build-system]
-requires = ["hatchling>=1.21.0"]
-build-backend = "hatchling.build"
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "dnspython"
 description = "DNS toolkit"
 authors = [{ name = "Bob Halley", email = "halley@dnspython.org" }]
-license = "ISC"
+license = {text = "ISC"}
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
@@ -29,6 +29,9 @@ requires-python = ">=3.8"
 dependencies = []
 dynamic = ["version"]
 
+[tool.setuptools.dynamic]
+version = {attr = "dns.version.version"}
+
 [project.optional-dependencies]
 dev = [
     "black>=23.1.0",
